### PR TITLE
[O11y][MySQL] Migrate `[Metrics MySQL] Database Overview` dashboard visualizations to lens in kibana 8.8.0

### LINKED ISSUE #7005: [O11y][MySQL] Update MySQL Metrics Dashboard
# Description

### **Task 1** 
Update panel positions and group related panels in the same row 
1. Connections
2. Statements, Query, Commands related panels
3. Buffer pool
4. Open tables & thread activity
5. Others such as network

### **Task 2**
Update title of **Rate of Questions** dashboard panel
**Old title**: Rate of Questions 
**New title**: Statements Executed

### **Task 3**
Move the row of connections as the first row

### **Task 4**
Remove the background colour for "Connection" Single metrics panel
### Screenshot 
image

### PR DESCRIPTION
**Pull request by rajvi-elastic**
- Enhancement

## What does this PR do?
- Migrate visualizations to lens for MySQL integration package

- Statistics for MySQL Lens migration:

<html>
<body>

MySQL dashboard | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Visualization | Lens | Visualization
[Metrics MySQL] Database Overview | 0 | 16 | 16 | 0
  | 0 | 16 | 16 | 0

</body>
</html>

## Checklist

- [X] I have added an entry to my package's `changelog.yml` file.
- [X] I have verified that panels are populated with data.
- [X] I have verified that panels are not distorted after being migrated to lens.
- [X] I have updated screenshots of dashboard.
- [X] I have verified that data count are matching and panel aggregations are same as before.

## Author's Checklist

- [X] Migrated panels should be removed from visualization folder.
- [X] Migrated visualizations are populating in current Kibana version 8.6.0 itself.

## Related issues

- Relates #5221 
- Closes #7005

Note:

The Kibana files and before migration dashboard image is not synced. Please find below screenshot to compare before and after migrated panels.
Metrics MySQL  Database Overview

### CODE REVIEW

**Review by milan-elastic — APPROVED** [review]

**Comment by agithomas** [comment]
Please share the below information 

What version of ES is used for `exports dashboard` ? Is it 8.8.x or 8.7.0 ?

**Comment by agithomas** [comment]
@SubhrataK , can you please review the new dashboard layout and suggest should something to be changed?

**Comment by rajvi-elastic** [comment]
> What version of ES is used for `exports dashboard` ? Is it 8.8.x or 8.7.0 ?

@agithomas Dashboards are exported from 8.8.0. But I am working on exporting dashboards from 8.7.0 as per today's discussion.

**Comment by rajvi-elastic** [comment]
@agithomas Both MySQL PRs ([Metrics PR](https://github.com/elastic/integrations/pull/5819), [Logs PR](https://github.com/elastic/integrations/pull/5818)) are updated with Dashboards exported from 8.7.0.

File: packages/mysql/img/metricbeat-mysql.png
Comment (agithomas) [inline packages/mysql/img/metricbeat-mysql.png:1]: What is this Rate of Questions dashboard panel ?

**Comment by agithomas** [comment]
Can we have some panel grouping here with related panels sharing the same row wherever possible?

1. Connections 
2. Statements & Query related panels
3. Buffer pool 
4. Open tables & thread activity
5. Others such as network

File: packages/mysql/img/metricbeat-mysql.png
Comment (rajvi-elastic) [reply packages/mysql/img/metricbeat-mysql.png:1]: This panels shows rate of `mysql.status.questions` field which represents **the number of statements executed by the server**.

**Review by agithomas — APPROVED** [review]
LGTM!

**Pull request merged**